draft-ietf-core-senml.md

```diff
@@ -1109,12 +1110,12 @@ for their review comments.
 
 # Links extension
 
-An extension to SenML to support links is expected to be registered and
-defined by {{I-D.ietf-core-links-json}}.
+An atribute to support an link extension for SenML is defined as a string
+attribute by this specification. The link extension can be used for additional
+information.  The definition and usage of the contents of this value are
```

Comment:
"Additional information about a SenML Record".
But I guess we need also base link for information about all Records.